src/test/scala/org/apache/spark/deploy/armada/ConfigSuite.scala

```diff
@@ -82,7 +76,8 @@ class ConfigSuite
       ("",            Map.empty[String, String]),
       ("a=1",         Map("a" -> "1")),
       ("a=1,b=2",     Map("a" -> "1", "b" -> "2")),
-      ("a=1,b=2,c=3", Map("a" -> "1", "b" -> "2", "c" -> "3"))
+      ("a=1,b=2,c=3", Map("a" -> "1", "b" -> "2", "c" -> "3")),
+      (" a=1,b= 2, c = 3 ", Map("a" -> "1", "b" -> "2", "c" -> "3"))
```

Comment:
If this is not desired, I can remove this.

Reply:
I think these are good cases to verify.  Also, looking at the validator code, I assume it would quickly flag the case of an accidental double-equals (e..g `"a=1, b==2"`) - should that be tested here?

Reply:
Any test case is worth adding.